google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/ConflictingAppEnginePluginCheck.java

```diff
@@ -109,6 +115,9 @@ public void hyperlinkUpdate(@NotNull Notification notification, @NotNull Hyperli
       String href = event.getDescription();
 
       if (DEACTIVATE_LINK_HREF.equals(href)) {
+        UsageTrackerProvider.getInstance()
+            .trackEvent(GctTracking.APP_ENGINE_OLD_PLUGIN_NOTIFICATION_CLICK)
```

Comment:
`APP_ENGINE_OLD_PLUGIN_NOTIFICATION_CLICK` tracks when a user clicks the link in the notification:

![](https://cloud.githubusercontent.com/assets/1039411/17418976/4716fa92-5a69-11e6-9acb-8043229c588b.png)

Clicking that link opens a dialog which prompts the user to uninstall the plugin (and optionally restart the application). That click is tracked by `APP_ENGINE_OLD_PLUGIN_DEACTIVATED`.